### ISSUE TITLE
Voice Over focus is not going "Profile image" while navigating via swipe gesture.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-iOS-SDK#708]
Category: Screen Reader
MAS Rule: Suggestion 
Repro Steps
1. Launch Embedded Social Test App.
2. Navigate to Hamburger button and double tap.
3. Navigate to hamburger button and double tap.
4. Navigate to the Profile link and double tap.
5. Navigate the page via swipe gesture.
6. Verify that voice over focus is going on Profile image or not.
Expected:
Voice Over focus should be  go on Profile image. And Profile image should be defined as interactive element. So that voice over focus go on it and screen reader user get the proper information.
Actual:
Voice Over focus is not going on Profile image. Profile image is not defined as interactive element.

Note:
1> Same issue is repro for icon image under the hamburger pane.
Test Environment:
OS: iOS 11.0.2
App: Embedded Social Test
App Version: 0.7.8 (8)
Tools: Voice Over, Keyboard
